src/settings-ui/Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -182,13 +182,10 @@
                                                 Margin="16,0,0,0"
                                                 CornerRadius="12">
                                             <Border.Background>
-                                                <SolidColorBrush Color="{ThemeResource SystemAccentColorLight3}"
+                                                <SolidColorBrush Color="{ThemeResource SystemAccentColor}"
                                                                  Opacity="0.3"/>
                                             </Border.Background>
-                                            <TextBlock Text="{x:Bind TargetApp}"
```

Comment:
The foreground must be set to White, not it will turn black in light mode and white in dark mode... We need to come up with a better design overall for the Apps token.

Reply:
> The foreground must be set to White, not it will turn black in light mode and white in dark mode...

If whe use white, then it'll be impossible to read when the accent color is getting light. I thought about this, and in my head it should work, given that the background color still has transparency.

> We need to come up with a better design overall for the Apps token.

Do agree with that. Maybe a grid? Better to do this together with #2027

Reply:
Yeah we need some sort of table design.. I'll check what the latest is on Token design guidelines.

Reply:
How does it look on your end? (Dark and light theme)

Reply:
![image](https://user-images.githubusercontent.com/9866362/183861465-dc25e1c7-7725-44f3-9055-4a5493886764.png)

SystemAccentColor should only be used for text labels (when inside of controls), not directly as a backdrop. This can result in a11y issues:

![image](https://user-images.githubusercontent.com/9866362/183861663-0fbc56f1-d4e9-4170-8beb-948656733b8d.png)

I'd be in favor in reverting this for now and creating a new issue and do a separate PR for redesigning the entire thing.

Reply:
Hm, those two screenshots look pretty clean. I don't see the problem in these?

Reply:
> SystemAccentColor should only be used for text labels (when inside of controls), not directly as a backdrop. This can result in a11y issues:

1. You use SystemAccentColor as the backdrop for the buttons in the ItemsControl. So...?
2. I know it can lead to [issues](https://github.com/microsoft/PowerToys/issues/16261), that's the reason I'm trying this :)

Reply:
The contrast between the background of the token and the row background will probably not meet the right standards. Either way, we need to update the whole listview and rethink where we place what and how 😄.